src/models/fhir/required.test.ts

```diff
@@ -50,6 +51,32 @@ describe("validateARTHealthCertData", () => {
       fhirHelper.hasRequiredFields(PdtTypes.Art, parseFhirBundle)
     ).not.toThrow();
   });
+
+  test("should throw error if ART healthcert lacks Observation.modality", () => {
+    let thrownError;
+    const malformedHealthCert = cloneDeep(exampleArtHealthCertWithNric);
```

Comment:
I think we can use the `examplePcrHealthCertWithNric` sample since PCRs do not have the Modality. Saves the trouble of finding and deleting the field.